src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -418,12 +418,10 @@
                     '{{ path("_wdt", { "token": "xxxxxx" }) }}'.replace(/xxxxxx/, newToken),
                     function(xhr, el) {
 
-                        /* Evaluate embedded scripts inside the toolbar */
-                        var i, scripts = [].slice.call(el.querySelectorAll('script'));
-
-                        for (i = 0; i < scripts.length; ++i) {
-                            eval(scripts[i].firstChild.nodeValue);
-                        }
+                        /* Evaluate in global scope scripts embedded inside the toolbar */
+                        eval.call({}, ([].slice.call(el.querySelectorAll('script')).map(function (script) {
+                          return script.firstChild.nodeValue;
+                        }).join('\n')));
```

Comment:
we should join them using `;\n` to avoid issues in case of having a script without a final semi-colon. The ASI rules might not always end a statement on a newline char.

Reply:
Thanks. Fixed in 09987102a8a92cdd2c6749edbe9c5709b5e14237